app/routers/event.py

```diff
@@ -251,3 +251,33 @@ def delete_event(event_id: int,
         # if the deletion is successful
     return RedirectResponse(
         url="/calendar", status_code=status.HTTP_200_OK)
+
+
+def is_date_before(start_time: datetime, end_time: datetime) -> bool:
+    """Check if the start_date is smaller then the end_time"""
+
```

Comment:
plese rempve blank line

Reply:
way? its looks like it's done in all author func...

Reply:
Although keeping the same styling as the rest of the file is usually the right thing to do, in this case [PEP257](https://www.python.org/dev/peps/pep-0257/#id16) standardize the convention:

> There's no blank line either before or after the docstring.

That's the case only for one-line docstrings.